clients/gtest/gemm_gtest.cpp

```diff
@@ -327,3 +340,9 @@ INSTANTIATE_TEST_CASE_P(checkin_blas3_tiny,
                         Combine(ValuesIn(tiny_matrix_size_range),
                                 ValuesIn(full_alpha_beta_range),
                                 ValuesIn(transA_transB_range)));
```

Comment:
Did you mean to use transA_transB_NN_range instead?

Reply:
The tiny should use all values NN, NT, TN, TT. Commit below removes the unused transA_transB_NN_range.